packages/ckeditor5-html-support/docs/features/general-html-support.md

```diff
@@ -214,6 +214,21 @@ The General HTML Support feature distinguishes several content types, each treat
 
 The enabled elements will not just be available "anywhere" in the content, as they still need to adhere to certain rules derived from the HTML schema and from common sense. Also, the behavior of specific types of elements in the editing area will be different. For instance, the object elements will only be selectable as a whole, and the inline elements will work the same as other formatting features supported by CKEditor 5 (e.g. bold, italic) do.
 
+### Enabling all elements and attributes
+
+In some cases, it might be desired to disable content filtering, so all elements and attributes will be allowed. It could be done with a special configuration:
```

Comment:
```suggestion
### Enabling all HTML features

It might be desired to enable all HTML features in some cases, so all elements and attributes will be allowed by the editor. It could be done with a special configuration:
```

It's a bit backward what we have in CKE4 and ACF, so in the case of CKE5, we are allowing for any HTML feature, not disabling filtration which is disabled by default.